object_database/web/cells_demo/conftest.py

```diff
@@ -0,0 +1,270 @@
+"""
```

Comment:
what's the difference between this and the stuff in CellsDemi_test? looks like duplicate code.